modify utm medium description (#320)
amplitude/amplitude-dev-center: includes/sdk-ts-browser/marketing-analytics-overview.md

|<div class="med-column">Name</div>|Description|
|-|-|
|`utm_source`| This identifies which website sent the traffic (for example, Google, Facebook) |

> |`utm_medium`| This identifies how the traffic gets  (for example, cpc, email) |

Reviewer: ```suggestion
|`utm_medium`| This identifies the type of campaign (for example, email, referral, or cpc) |
```
